.github/workflows/pre-submit.actions.yml

```
@@ -110,7 +110,7 @@ jobs:
       - name: Verify version references
         if: ${{ contains(github.event.pull_request.body, '#label:release') }}
         run: |
-          echo ${{ github.event.pull_request.body }} | awk '{print $2}'
+          RELEASE_TAG=$(echo "${{ github.event.pull_request.body }}" | grep -o -e 'v[0-9].[0-9].[0-9]')
```

'.' will get interpreted as regex so it probably should be escaped. otherwise it matches strings like 'v1a2b3'.
```suggestion
          RELEASE_TAG=$(echo "${{ github.event.pull_request.body }}" | grep -o -e 'v[0-9]\.[0-9]\.[0-9]')
```

---

Noted, I think it got copied into https://github.com/slsa-framework/slsa-github-generator/pull/1210/files, let's merge yours which does a code freeze, but I'll fix that next release